server/lib/renderers/pdf.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @description 
+ * This service is responsible for producing PDFs on the server, it accepts handlebar templates and uses the html
+ * renderer to produce valid HTML - then streaming this through the wkhtmltopdf application to produce PDFs.
+ * 
+ * @requires wkhtmltopdf
+ * @requires stream-to-promise
+ * @requires path
+ * @requires q
+ */
+const wkhtmltopdf     = require('wkhtmltopdf');
+const q               = require('q');
+const streamToPromise = require('stream-to-promise');
+const process         = require('process');
```

Comment:
Didn't realize you could `require()` process.  Interesting!  Could you list it in the `@requires` block?